bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/i18n/boschshc.properties

```diff
@@ -10,3 +10,7 @@ offline.conf-error-ssl = The SSL connection to the Bosch Smart Home Controller i
 offline.long-polling-failed.http-client-null = Long polling failed and could not be restarted because http client is null.
 offline.long-polling-failed.trying-to-reconnect = Long polling failed, will try to reconnect.
 offline.interrupted = Conneting to Bosch Smart Home Controller was interrupted.
+
+offline.conf-error.empty-device-id = No device ID set.
+
+offline.conf-error.invalid-device-id = Device ID is invalid.
```

Comment:
In the last openhab PR an empty line at EOF was requested.

Reply:
It's strange, there is an empty line in my local code and no git changes are detected. Could you check your locally checked out code?